### DISCUSSION TITLE
Welcome to BootstrapBlazor.Geolocation Discussions!

### DISCUSSION THREAD

**Discussion by densen2014** [BootstrapBlazor/BootstrapBlazor.Geolocation#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tavanuka** [comment]
Hi, my name is Mark, and go by the name tavanuka. I am an apprentice developer for a company and learning ASP.NET/Blazor side of C#. I found this repository not too long ago and found it a wonderful implementation of geolocation.
